modules/serverless-ossm-disabling-network-policies.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * /serverless/integrations/serverless-ossm-setup.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-ossm-disabling-network-policies.adoc_{context}"]
+= Disabling the default network policies 
+
+To disable the default network policy generation, you can annotate the `KnativeEventing` and `KnativeServing` custom resources (CRs) with the `serverless.openshift.io/disable-istio-net-policies-generation` annotation.
+
+This prevents the operator from applying the default network policies.
+
+.Prerequisites
+
+* You have installed the {ServerlessOperatorName} and Knative Serving and Knative Eventing on your cluster.
+* You have installed {SMProductName} with the mTLS functionality enabled.
+
+* You have cluster administrator permissions on {ocp-product-title}, or you have cluster or dedicated administrator permissions on {rosa-product-title} or {dedicated-product-title}.
+
+* Install the OpenShift CLI (`oc`).
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads.
```

Comment:
```suggestion
* You have access to a project with the appropriate roles and permissions to create applications and other workloads.
```
If they have an access to a project, then they have created or arranged a project with roles and perms, so it can be removed.